src/mbgl/renderer/symbol_bucket.hpp

```diff
@@ -46,25 +47,35 @@ class SymbolBucket : public Bucket {
 private:
     friend class SymbolLayout;
 
-    typedef ElementGroup<2> TextElementGroup;
-    typedef ElementGroup<4> IconElementGroup;
-    typedef ElementGroup<1> CollisionBoxElementGroup;
+    typedef gl::ElementGroup<2> TextElementGroup;
+    typedef gl::ElementGroup<4> IconElementGroup;
+    typedef gl::ElementGroup<1> CollisionBoxElementGroup;
 
     struct TextBuffer {
-        TextVertexBuffer vertices;
-        TriangleElementsBuffer triangles;
+        std::vector<TextureRectVertex> vertexes;
```

Comment:
For `indexes` I actually meant to use `lines` or `triangles` depending on the primitive. I'll switch back to `vertices`.